tests/config_files/config_testing.py

```diff
@@ -155,7 +155,7 @@ def testSingleN200Connected(self):
         self.assertEqual(options.tx_main_antennas, [0])
 
     def testSingleN200BadMain(self):
-        """ Single N200, connected to invalid main antenna """
+        """ Single N200, connected to invalid main antenna (main_antenna_count = 16) """
```

Comment:
Probably not important now, but what if a radar has a different number of main antennas than 16?

Reply:
This test class modifies the file `base_config.ini`, which has `main_antenna_count = 16`. If someone makes a config file with a different value for this, it will test against that value instead.